packages/store/internals/src/initial-state.ts

```diff
@@ -0,0 +1,17 @@
+import { InjectionToken } from '@angular/core';
+
+export const INITIAL_STATE_TOKEN = new InjectionToken<any>('INITIAL_STATE_TOKEN');
+
+export class InitialState {
+  private static value = {};
```

Comment:
private static state: ObjectKeyMap<any> = {};

Reply:
The `ObjectKeyMap` type is not available form the internals package

Reply:
we could add it `@ngxs/store/commons`